jira/client.py

```diff
@@ -827,6 +830,16 @@ def _get_items_from_page(
             # improving the error text so we know why it happened
             raise KeyError(str(e) + " : " + json.dumps(resource))
 
+    def _get_batch_size(self, item_type: Type[ResourceType]) -> Optional[int]:
+        batch_sizes: Dict[Type[Resource], Optional[int]] = self._options[
+            "default_batch_size"
+        ]
+        try:
+            item_type_batch_size = batch_sizes[item_type]
+        except KeyError:
+            item_type_batch_size = batch_sizes.get(Resource, None)
```

Comment:
should be supplemented in the docstring of this function
```python
"""
Returns the batch size for the given resource type from the options.

Returns:
   Optional[int]: The batch size to use. When the configured batch size is None, the batch size is determined by the Jira backend.
"""
```


May be worth also a quick inline comment

```suggestion
            # None to let the Jira backend determine the batch size
            item_type_batch_size = batch_sizes.get(Resource, None)
```